script/lib/install-application.js

```diff
@@ -67,7 +67,7 @@ module.exports = function (packagedAppPath) {
     const desktopEntryTemplate = fs.readFileSync(path.join(CONFIG.repositoryRootPath, 'resources', 'linux', 'atom.desktop.in'))
     const desktopEntryContents = template(desktopEntryTemplate)({
       appName, appFileName: atomExecutableName, description: appDescription,
-      installDir: '/usr', iconPath
+      installDir: userLocalDirPath, iconPath
```

Comment:
This line actually changes the default here from being /usr to /usr/local - is that intentional?